specification/cognitiveservices/data-plane/SpellCheck/V1.0/SpellCheck.json

```diff
@@ -0,0 +1,468 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Spell Check API - v7",
+    "description": "The Spell Check API lets you check a text string for spelling and grammar errors.",
+    "version": "1.0"
+  },
+  "parameters": {
+    "x-bingapis-sdk": {
+      "name": "X-BingApis-SDK",
+      "description": "Activate swagger compliance",
+      "x-ms-parameter-location": "method",
+      "required": true,
+      "type": "string",
+      "in": "header",
+      "x-ms-enum": {
+        "name": "XBingApisSDK",
+        "modelAsString": true
+      },
+      "enum": [
+        "true"
+      ]
+    }
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/spellcheck": {
+      "post": {
+        "summary": "The Bing Spell Check API lets you perform contextual grammar and spell checking. Bing has developed a web-based spell-checker that leverages machine learning and statistical machine translation to dynamically train a constantly evolving and highly contextual algorithm. The spell-checker is based on a massive corpus of web searches and documents.",
+        "operationId": "SpellCheck",
+        "tags": [
+          "SpellCheck"
+        ],
+        "consumes": [
+          "application/x-www-form-urlencoded"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/x-bingapis-sdk"
+          },
+          {
+            "name": "Accept",
```

Comment:
It's awesome you support JSON-LD, but Autorest does not, so this parameter cannot be used by Autorest and Swagger in general to deserialize a model.
I would suggest to remove it, this has no chance to work in SDK :/